src/QMCWaveFunctions/Fermion/DiracMatrixComputeCUDA.hpp

```diff
@@ -159,23 +153,21 @@ class DiracMatrixComputeCUDA : public Resource
    *  \param[out]     log_values  log determinant value for each matrix, batch_size = log_values.size()
    *
    */
-  inline void mw_computeInvertAndLog(CUDALinearAlgebraHandles& cuda_handles,
-                                     DualVector<VALUE_FP>& psi_Ms,
-                                     DualVector<VALUE_FP>& inv_Ms,
-                                     const int n,
-                                     const int lda,
-                                     DualVector<LogValue>& log_values)
+  inline void mw_computeInvertAndLog_stride(CUDALinearAlgebraHandles& cuda_handles,
```

Comment:
so if we are always taking the hit and using the psiM_fp_ memory I think we can simplify and just use this method.  Then we also have a continuous transfer instead of a transfer per psiM

Reply:
This code path needs some revising. Right now, it transfers mat_inv_FP to host, copy to mat_inv_MP and then transfer back to device. To many round trip here. We need a separate discussion about how to adjust all these transfer.